crates/flox-rust-sdk/src/models/environment_ref.rs

```diff
@@ -1,479 +1,112 @@
-use std::io::{self, ErrorKind};
```

Comment:
I think we'll still need the logic to handle defaulting named environments for `-e`, but I don't think we'll support multiple environments in a `.flox`. So I'm not sure we want to drop all the named handling, and I'm also not sure the way this implements `.flox` handling is correct

Reply:
My understanding was that we drop the "named" envs, but that .flox can either be a local env with its definitions in place, or a remote env with its staged definition managed centrally (within a floxmeta clone) subject to sync with floxhub.

Thus you dont access "remote" envs directly, but by way of linking it to your current dir (+ .flox).
That kinda makes more sense when we'd allow multiple envs per .flox, but i guess that's a good point to circle around.